gallery/psget/module/RequireLicenseAcceptance.md

```diff
@@ -0,0 +1,213 @@
+```
+
+### Example 8: Install Script with dependencies requiring license acceptance and -AcceptLicense
+Script 'ScriptRequireLicenseAcceptance' depends on module 'ModuleRequireLicenseAcceptance'. User is not prompted to accept license as -AcceptLicense is specified.
+```PowerShell
+PS C:\> Install-Script -Name ScriptRequireLicenseAcceptance -AcceptLicense
+```
+
+### Example 9: Install module requiring license acceptance on a client older than PSGetFormatVersion 2.0
+```PowerShell
+PS C:\windows\system32> Install-Module -Name ModuleRequireLicenseAcceptance
+
+WARNING: The specified module 'ModuleRequireLicenseAcceptance' with PowerShellGetFormatVersion '2.0' is not supported by the current version of PowerShellGet. Get the latest version of the PowerShellGet module to install this module, 'ModuleRequireLicenseAcceptance'.
+
+```
+### Example 10: Save Module requiring license acceptance
+```PowerShell
+PS C:\> Save-Module -Name ModuleRequireLicenseAcceptance -Path C:\Saved
+
+License Acceptance
+
+License 2.0
+Copyright (c) 2016 PowerShell Team
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software.
+
+Do you accept the license terms for module 'ModuleRequireLicenseAcceptance'.
+[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): 
+```
+This command shows the license from license.txt file and prompts the user to accept the license.
+
+### Example 11: Save Module requiring license acceptance with -AcceptLicense
+```PowerShell
+PS C:\> Save-Module -Name ModuleRequireLicenseAcceptance -AcceptLicense -Path C:\Saved
+```
+Module is saved without any prompt to accept license.
+
+### Example 12: Update Module requiring license acceptance
+```PowerShell
+PS C:\> Update-Module -Name ModuleRequireLicenseAcceptance
+
+License Acceptance
+
+License 2.0
+Copyright (c) 2016 PowerShell Team
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software.
+
+Do you accept the license terms for module 'ModuleRequireLicenseAcceptance'.
+[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): 
+```
+This command shows the license from license.txt file and prompts the user to accept the license.
+
+### Example 13: Update Module requiring license acceptance with -AcceptLicense
+```PowerShell
+PS C:\> Update-Module -Name ModuleRequireLicenseAcceptance -AcceptLicense
+```
+Module is updated without any prompt to accept license.
```

Comment:
Add a reference to the gallery UI doc on this.

Reply:
Done

---
In reply to: [136225643](https://github.com/PowerShell/PowerShell-Docs/pull/1620#discussion_r136225643) [](ancestors = 136225643)